base/acme/src/main/java/org/dogtagpki/acme/server/ACMEEngine.java

```diff
@@ -192,7 +193,29 @@ public URL getBaseURL() {
         return config.getBaseURL();
     }
 
-    public void loadConfig(String filename) throws Exception {
+    public void loadLoggingProperties(String loggingProperties) throws Exception {
+
+        File file = new File(loggingProperties);
+        if (!file.exists()) return;
+
+        logger.info("Loading " + loggingProperties);
+        Properties properties = new Properties();
+        properties.load(new FileReader(file));
+
+        for (String key : properties.stringPropertyNames()) {
+            String value = properties.getProperty(key);
+
+            logger.info("- " + key + ": " + value);
+            if (!key.endsWith(".level")) continue;
```

Comment:
Since we call the file `logging.properties` I am expecting that the provided values should overwrite the existing to modify the logs. So I would try to update the full [configuration](https://docs.oracle.com/en/java/javase/21/docs/api/java.logging/java/util/logging/LogManager.html#updateConfiguration(java.io.InputStream,java.util.function.Function))

Reply:
OK, I can try that. I wasn't sure whether the `LogManager` will override just the params specified in the custom `logging.properties` or replace the entire logging config with the custom `logging.properties` (meaning the handlers need to be redefined in the custom `logging.properties`).

Reply:
I guess there was no way to have one copy of this?

Reply:
We aren't really duplicating the `logging.properties` here. The default `logging.properties` is a read-only file owned by the package and it contains the full default logging config. The custom `logging.properties` is owned by the user and is only needed if you want to override the default config, and right now you can only change the `level`, so it will be much simpler than the default `logging.properties`.

Also, we probably don't want to rely on just a single `logging.properties` owned by the user since it would be more difficult to change the default config if needed. We would need to write an upgrade script to parse the user's config and there's a risk that our changes could conflict with user's customization.